task-list/app/views/tasks/edit_form.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Edit Task</h1>
+
+<%= form_for @task, url: { action: 'update' }, method: :patch do |f| %>
+  <%= render partial: 'form', locals: { f: f } %>
```

Comment:
Innovative use of a partial within a `form_for` tag! This definitely alleviates some of the work that other students needed in order for the form to know what url/method it should be using.